# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iwlwifi/depmod.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iwlwifi-0.0.18/Makefile.vanilla	2007-06-14 18:47:19.000000000 +0000
+++ iwlwifi-0.0.18/Makefile	2007-06-14 18:47:25.000000000 +0000
@@ -108,7 +108,7 @@
 install: $(addprefix $(DIR),$(addsuffix .ko,$(list-m)))
 	@install -d $(KMISC)
 	@install -m 644 -c $^ $(KMISC)
-	@/sbin/depmod -a ${KVER}
+	#/sbin/depmod -a ${KVER}
 	@echo -e "\nModule " $^ "(s) installed into:\n\t$(KMISC)"
 	@echo -e "\nDon't forget to copy firmware to your hotplug's firmware directory "
 	@echo "and have the hotplug tools in place."
